src/screens/Bridge/Exchange-CrossChain/CrossChain.tsx

```diff
@@ -179,7 +174,7 @@ function CrossChain({ fetchCrossChainTitle }: Props) {
     <SendHighGasModal value={fromValue} contact={fromAddress} chain={chain} txFeeInfo={feeInfo} />
   ) : null;
 
-  const errorMessage = estimationErrorMessage | notEnoughForFeeErrorMessage;
+  const errorMessage = estimationErrorMessage | transactionFee?.errorMessage;
```

Comment:
Do you know if this returns a friendly error message or are we just returning the message from the server here?